integrationtests/src/test/java/io/kroxylicious/proxy/multitenant/MultiTenantIT.java

```diff
@@ -85,7 +82,7 @@
 @ExtendWith(KafkaClusterExtension.class)
 public class MultiTenantIT {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(MultiTenantIT.class);
+    public static final Logger LOGGER = LoggerFactory.getLogger(MultiTenantIT.class);
```

Comment:
It does not, this one was quite interesting too. The logger in MultiTenantIT is unused, but tests failed when I removed it with error.

```
java.util.ServiceConfigurationError: java.net.spi.InetAddressResolverProvider: Provider io.kroxylicious.net.IntegrationTestInetAddressResolverProvider could not be instantiated
... snip ...
Caused by: java.lang.NullPointerException: Cannot invoke "org.slf4j.Logger.info(String)" because "io.kroxylicious.net.IntegrationTestInetAddressResolverProvider.LOGGER" is null
	... snip ...
	... 153 more
	```
	
I guess something bad happens if the first call to `LoggerFactory.getLogger` is part of the service loading.

Reply:
Urgh. Might need to look into that a bit harder.